core/src/main/resources/hudson/model/UsageStatistics/global.groovy

```diff
@@ -8,7 +8,7 @@ def f=namespace(lib.FormTagLib)
 
 f.section(title: _("Usage Statistics")) {
     if (UsageStatistics.DISABLED) {
-        span(class: "jenkins-not-applicable") {
+        div(class: "jenkins-not-applicable jenkins-!-margin-bottom-2") {
```

Comment:
Didn't look great as it was touching the line

Before:
<img width="719" alt="image" src="https://github.com/user-attachments/assets/c592ec6d-5eec-4ab9-bdce-ddb54a531f5c">

After:
<img width="711" alt="image" src="https://github.com/user-attachments/assets/bdb3a32b-d0a6-415e-8e52-24653909569e">